src/impl/apis/data_api_impl.py

```diff
@@ -258,35 +259,152 @@ async def get_time_range(
     return time_range
 
 
+async def get_query_attributes(query_parameters, token_model, pool):
+    """
+    Gets the metadata + query parameters
+    """
+    category = await get_category(query_parameters.category_id, token_model=token_model, pool=pool)
+    if category.type not in ["single_location", "flow"]:
+        raise HTTPException(
+            status_code=HTTPStatus.NOT_FOUND,
+            detail="Could not find category",
+        )
+    base_table_name = f"{category.type}_data"
```

Comment:
Annoying to not be able to add this as an attr on the Category objects really, ho hum.